coreneuron/io/nrn_checkpoint.cpp

```diff
@@ -25,120 +25,46 @@
 #include "coreneuron/utils/nrnoc_aux.hpp"
 
 namespace coreneuron {
-bool nrn_checkpoint_arg_exists;
 bool _nrn_skip_initmodel;
 }  // namespace coreneuron
-#define UseFileHandlerWrap 0
 
-#if UseFileHandlerWrap
```

Comment:
@alkino : can you remind me how this `FileHandlerWrap` was helping? it was ASCII based and one can look at the values? If so, how this could be done in new implementation?

Reply:
I have no idea, but it was not use, so I remove.

Reply:
for now, its ok. if needed, we will update the implementation.